src/raftstore/store/clocktime.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright 2016 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+pub use self::inner::now_monotonic_raw_clocktime;
+
+#[cfg(any(target_os = "macos"))]
+mod inner {
+    use time::{self, Timespec};
+
+    pub fn now_monotonic_raw_clocktime() -> Timespec {
+        // TODO Add monotonic raw clock time impl for macos
+        // Currently use the `time::get_time()` instead.
+        time::get_time()
+    }
+}
+
+#[cfg(not(any(target_os = "macos")))]
+mod inner {
+    use std::io;
+    use time::Timespec;
+    use libc;
+
+    pub fn now_monotonic_raw_clocktime() -> Timespec {
+        let mut t = libc::timespec {
+            tv_sec: 0,
+            tv_nsec: 0,
+        };
+        let res = unsafe { libc::clock_gettime(libc::CLOCK_MONOTONIC_RAW, &mut t) };
```

Comment:
`Instant::now()` is implemetated using `CLOCK_MONOTONIC`. And it could be affect by NTP and it could rollback. 
For leader lease, the clocktime needs to be accurate. It doesn't make sense if the clocktime rollbacks. So `CLOCK_MONOTONIC_RAW` is better suited.